machAeroTutorials/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
So, the idea is to avoid the `cgns_utils` for at least airfoil mesh convergence study. Prefoil is one of the option to increase the airfoil surface nodes using the polynomial interpolation. I see, it sounds like use prefoil for this. I can rephrase the sentences and provide prefoil as an option (Which can be helpful for people who already have access to it).

I thought `.gitignore` supposed not to include that. I will revert this file in the commit